src/main/java/bio/terra/service/filedata/flight/ingest/IngestFilePrimaryDataStep.java

```diff
@@ -29,6 +29,19 @@ public IngestFilePrimaryDataStep(Dataset dataset,
         this.dataset = dataset;
     }
 
+    private GoogleBucketResource getBufferInfo(FlightContext context) {
```

Comment:
- Should this be renamed to "getBucketInfo"?
- It looks like the same method was added to IngestPopulateFileStateFromFileStep.java, so maybe this could be pulled out into a shared util?

Reply:
I was confused about the name too! I'm guessing it was just a 🎤 issue. I agree on pulling in into utils.

Reply:
Haha,  totally a 🎤 issue.

Reply:
I did not want to mint a new module for this one method it will be going away when we get rid of the single file load endpoint. I think that will happen in the next two sprints